src/app/app.config.ts

```diff
@@ -0,0 +1,9 @@
+import { ApplicationConfig } from '@angular/core';
```

Comment:
the same

Reply:
These files were created when i just create a project.When i said that i don't have app module they were there.I tried to delete it but app was broken.Maube it's beacuse i launched this app on angular 17